### ISSUE TITLE
Uninstaller

### ISSUE THREAD

**Issue by nevadascout** [nevadascout/Dash#72]
If (for some bizarre reason) you want to uninstall dash, there isn't an uninstaller to do so.

This isn't a problem now as you can just delete the Dash folder in your C drive, however once Dash is registered for file types, we will need a proper uninstaller.

It would also be nice if Dash could appear in the "Programs & Features" section of the control panel, with an option to uninstall from there.

**Comment by trdwll** [comment]
Instead of making a custom uninstaller, you can use http://www.jrsoftware.org/isinfo.php to handle the installing and uninstalling.

**Comment by nevadascout** [comment]
Yeah, I was looking at that and InstallShield, but decided in the end to use the .NET ClickOnce system for future installs/uninstalls. Much easier to implement, plus we can also sign the code as well.
